tests/end-to-end/external-solution-integration/internal/scenario/event_mesh/wrapper_counter_pod.go

```diff
@@ -0,0 +1,22 @@
+package event_mesh
+
+import (
+	"time"
+
+	retrygo "github.com/avast/retry-go"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/testkit"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/testsuite"
+)
+
+const retryAttemptsCount = 240
```

Comment:
I've used the same constants as in the most deeper retry component used, see "pkg/retry/retry.go":
```
const retryAttemptsCount = 120
const retryDelay = 1 * time.Second
```
to be more clear that it will overwrite them